tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -145,15 +144,18 @@ static void test_advx_main(void)
 
 	k_sleep(K_MSEC(1000));
 
-	printk("Starting non-conn non-scan with aux 1M advertising...");
-	err = ll_adv_aux_ad_data_set(handle, AD_OP, AD_FRAG_PREF,
-				     sizeof(adv_data), (void *)adv_data);
+	printk("Enabling non-conn non-scan without aux 1M advertising...");
+	err = ll_adv_enable(handle, 1);
 	if (err) {
 		goto exit;
 	}
+	printk("success.\n");
```

Comment:
we should verify that advertising indeed has started i.e. sending packets over the air. We can then also remove the following sleep

Reply:
Is there a bsim API to synchronize out of band between devices?

Reply:
Good question, a quick search didn't show one

Reply:
Checking air traffic can be done by having the test script check the logs after the fact. Alternatively, EDTT coordinates two or more devices and can check whether for example advertisements from one device are seen by another device. Maybe something like that would be the way to go here?